<?xml version="1.0"?>

<launch>
	<node pkg="subt_rl" type="pub_gazebo_tf.py" name="pub_gazebo_tf" output="screen" />
	<node pkg="subt_rl" type="goal_uwb.py" name="goal_uwb" output="screen" />
</launch>
